### PR TITLE
Docs: Configuration automation via script console

### PR DESCRIPTION
**Pull request by samrocketman**
This change comes from a request in a JIRA comment.

https://wiki.jenkins.io/display/JENKINS/JIRA+Steps+Plugin?focusedCommentId=122815139#comment-122815139

Note: this is not able to be run from a pipeline.  It can only run by a Jenkins administrator via the [script console][1].

[1]: https://wiki.jenkins.io/display/JENKINS/Jenkins+Script+Console

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -57,4 +57,65 @@ Get started by [installing](install) this plugin and also refer to the [Release
 
 **NOTE:** The documentation is always up to date with latest version (or master). If you are on old version please refer to the code.
 
+## Automate configuring via Jenkins Script Console
```
Comment (nrayapati) [inline docs/index.md]: [config](https://github.com/jenkinsci/jira-steps-plugin/blob/master/docs/pages/general/config.md) is the right place for this, would you mind moving this section there. Thanks.

**Comment by nrayapati** [comment]
@samrocketman Thanks for the pull request I appreciate it.

File: docs/index.md (hunk 1 above)
Comment (samrocketman) [inline docs/index.md]: Sure I'll move it.

**Comment by samrocketman** [comment]
Amended my last commit and addressed your comment.  Please review again, @nrayapati.

**Comment by nrayapati** [comment]
Looks good.

**Comment by neybar** [comment]
Thanks for the example!

**Comment by samrocketman** [comment]
If you're going to be at Jenkins world I'm presenting on the script console there.

**Comment by nrayapati** [comment]
Awesome @samrocketman, All the best. 

I am not going though.

**Pull request merged**